lib/src/executor/runtime_host.rs

```diff
@@ -396,6 +396,7 @@ pub enum ErrorDetail {
     },
     /// Size of the logs generated by the runtime exceeds the limit.
     LogsTooLong,
+    ForbiddenHostCall,
```

Comment:
Please add one error in `validate.rs`, one in `header_body.rs`, etc. instead of here